src/Category/Monad.agda

```diff
@@ -26,6 +26,8 @@ RawMonadZero M = RawIMonadZero {I = ⊤} (λ _ _ → M)
 RawMonadPlus : ∀ {f} → (Set f → Set f) → Set _
 RawMonadPlus M = RawIMonadPlus {I = ⊤} (λ _ _ → M)
 
+module RawMonadT = RawIMonadT
+
```

Comment:
Is it worth restricting the type of the parameter here, so that we can't use `module RawMonadT` on an indexed one? So, `module RawMonadT {f} {T} (MonT : RawMonadT {f} T) = RawIMonadT MonT`.

Reply:
That is probably wise, else the results might be surprising.

Reply:
I started with the restricted type but doing so leads to unsolved meta
variables in other parts of the PR.